__tests__/utils.tsx

```diff
@@ -246,7 +233,7 @@ describe('getPathInfo()', () => {
       test.each([{}, { data: { uuid: {} } }])(
         'response = %p',
         async (response) => {
-          mockHttpPost(apiEndpoint, returnJson(response))
+          mockApi(apiEndpoint, returnJson(response))
```

Comment:
This will not work. Add a function `apiToReturnEmptyJson()` which returns an empty dictionary.